Marlin/src/pins/mega/pins_MIGHTYBOARD_REVE.h

```diff
@@ -203,6 +203,7 @@
 #define LED_PIN                               13  // B7
 #define CUTOFF_RESET_PIN                      16  // H1
 #define CUTOFF_TEST_PIN                       17  // H0
+#define CUTOFF_SR_CHECK_PIN                   70  // G4 (TOSC1)
```

Comment:
Like so ☝️

Reply:
Yep but that pin definition are from an unofficial core fix from #15607
Which I have implemented in a new branch with 80pins 
https://github.com/chepo92/ArduinoCore-avr/tree/80pins1280-2560
As those 80 pins are not mapped in the official core (just 70), I raised the issue there
https://github.com/arduino/ArduinoCore-avr/issues/330
Which lead me to an unofficial core with 100 pins defined differently, but maybe not supporting all libraries
https://github.com/MCUdude/MegaCore/blob/master/avr/variants/100-pin-arduino-mega/pins_arduino.h

So i'm wondering if should make a board variant "Mightyboard" (or Makerboard hahaha), synced with the official core, as there exist the "Sanguino" which is the Anet board and derivatives.

Sorry for long comentary, but this will be useful for the record

Reply:
Marlin maps all the ATmega pins via FastIO. In `fastio_1280.h` pin 70 is mapped via…
```cpp
#define DIO70_PIN   PING4
#define DIO70_RPORT PING
#define DIO70_WPORT PORTG
#define DIO70_DDR   DDRG
#define DIO70_PWM   nullptr
```

Reply:
Marlin maps all the ATmega pins via FastIO. In `fastio_1280.h` it currently maps up to pin 85. Adding more pins shouldn't be too hard. Does it need to be a new fastio file? It can just add onto `fastio_1280.h` based on some other flag.